### PR TITLE
New Function to Show BIS Style Curator Messages

### PR DESCRIPTION
**Pull request by 654wak654**
**When merged this pull request will:**
- Add a function for zeus messages and change all displayTextStructured calls to it. New function uses `BIS_fnc_showCuratorFeedbackMessage` for zeus messages, which shows a slightly fancier message on the top bar of the zeus ui.

### CODE REVIEW

**Comment by BaerMitUmlaut** [comment]
The variant with format is not documented and due to the params probably also not functional.

**Comment by 654wak654** [comment]
> The variant with format is not documented and due to the params probably also not functional.
Oh yes, will add that. It is functional, common's displayTextStructured works the same way.

File: addons/zeus/functions/fnc_showMessage.sqf [hunk 1]
```diff
@@ -0,0 +1,30 @@
+/*
+ * Author: 654wak654
+ * Shows a zeus message through the BIS function, handles localization
```
Comment (jonpas) [inline addons/zeus/functions/fnc_showMessage.sqf]: Missing full stop.

Comment (654wak654) [inline addons/zeus/functions/fnc_showMessage.sqf]: Most headers doesn't have dots there?

Comment (jonpas) [inline addons/zeus/functions/fnc_showMessage.sqf]: They should. :)

**Comment by BaerMitUmlaut** [comment]
You then take an array within an array as a parameter, without a second parameter. I think just passing what you would pass `format` might be a little nicer.

**Pull request merged**